tests/core/test_integration.py

```diff
@@ -1586,17 +1590,7 @@ def _validate_apply(context):
 @pytest.mark.parametrize(
     "from_, to",
     [
-        (ModelKindName.INCREMENTAL_BY_TIME_RANGE, ModelKindName.VIEW),
-        (ModelKindName.INCREMENTAL_BY_TIME_RANGE, ModelKindName.EMBEDDED),
         (ModelKindName.INCREMENTAL_BY_TIME_RANGE, ModelKindName.FULL),
-        (ModelKindName.VIEW, ModelKindName.EMBEDDED),
```

Comment:
All of these transitions became invalid for this test now that the `FULL` and `EMBEDDED` kinds are disabled for Python models.

This test is tightly coupled with the `sushi.items` model which is a Python model